component-library/component-lib/src/lib/jl-checkbox/jl-checkbox.component.ts

```diff
@@ -1,10 +1,12 @@
-import { Component, forwardRef, Input } from '@angular/core';
-import { ControlValueAccessor, FormControl, FormGroup, NG_VALUE_ACCESSOR, ValidatorFn, Validators } from '@angular/forms';
+import {Component, forwardRef, Input, OnInit} from '@angular/core';
+import {ControlValueAccessor, FormControl, FormGroup, NG_VALUE_ACCESSOR, ValidatorFn} from '@angular/forms';
+import {DSSizes} from "../../shared/constants/jl-components/jl-components.constants/jl-components.constants";
 
 export interface ICheckBoxComponentConfig {
   formGroup: FormGroup;
+  size?: keyof typeof DSSizes | DSSizes;
   error?: true;
-  small?: true; //DS Default is Large, hence this being changed for consistency 
+  small?: true; //DS Default is Large, hence this being changed for consistency
   mixed?: true;
   disableFocus?: boolean; //Default is true
   checked?: boolean; //Controls the actual checked state of the component
```

Comment:
Is this used anywhere? It can probably go since it's controlled by the formControl